README.md

```diff
@@ -28,6 +33,12 @@ Then you can find the `osdctl` binary file in the `./dist` subdirectory matching
 
 Release are available on Github
 
+### Creating a release
+
+Repository owners can create a new `osdctl` release with the `make release` target. An API token with `repo` permissions is required. [See: https://goreleaser.com/environment/#api-tokens](https://goreleaser.com/environment/#api-tokens)
```

Comment:
I've found that I need to manually run `go tag v0.y.z`. Should that be added here?

Reply:
I think you meant `git tag v0.y.z`, and yes, good call.  It has to have a tag to use.  I'll update the docs.

Reply:
yes git tag